schedule/filesystem/xfstests_micro_ppc.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: 'xfstests@sle_micro_ppc'
+description:    >
+  Maintainer: An Long <lan@suse.com>
+  xfstests test suite for SLE-Micro on ppc
+schedule:
+    - installation/bootloader
+    - xfstests/partition
+    - xfstests/run
+    - xfstests/generate_report
```

Comment:
Seems need to remove this line, because we trigger generate_report in run.pm now.

Reply:
Thanks for your catch! removed